scripts/get_akse.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
+
+# Copyright Jorge Palma.
```

Comment:
We're supposed to have the Microsoft standard copyright header on each code file, which I think would include scripts as well, although we haven't been doing that. I'm not a lawyer, but I think asserting a different copyright might be problematic.

Reply:
Yup, copy pasta foo. I technically think it should be alright but def not gonna dive into that. Changed.